packages/block-library/src/embed/edit.js

```diff
@@ -279,6 +294,7 @@ const EmbedEdit = ( props ) => {
 					icon={ icon }
 					label={ label }
 					insertBlocksAfter={ insertBlocksAfter }
+					clientId={ clientId }
```

Comment:
Interesting that we need to pass the block id so the [caption component can determine](https://github.com/WordPress/gutenberg/blob/80465ea0ef7f804e515bb1b0137a31890bd7baaf/packages/block-editor/src/components/block-caption/index.native.js#L50-L56) if it should display itself. Nothing to touch in this PR but, I intuitively feel that displaying the caption or not should be the caption's parent responsibility, and in this case only governed by the `isSelected`. Anyway, just a comment for the time being since it caught my attention.

Reply:
Yeah, I'm surprised too that the `BlockCaption` component requires the `clientId` to determine if the block is selected, I agree that looks more intuitive if we had a `isSelected` prop here instead `clientId`.

Reply:
> ... if we had a isSelected prop here instead clientId.

Hehe, we do have iSelected passed, 4 lines above this one :giggle:

And just to stay clear, let's leave it as is for this PR.